### PR TITLE
remove user deletion from LDAP sync

### PR DESCRIPTION
**Pull request by ntarocco**

### CODE REVIEW

File: cds_ils/ldap/api.py [hunk 1]
```diff
@@ -375,6 +364,68 @@ def sync_users():
...
+
+    # get all Invenio remote accounts and prepare a list with needed info
+    invenio_users = []
+    for remote_account in remote_accounts:
+        invenio_users.append(
+            dict(
+                remote_account_id=remote_account.id,
+                remote_account_person_id=remote_account.extra_data[
+                    "person_id"
+                ],
```
Comment (topless) [inline cds_ils/ldap/api.py:399]: For consistency
```suggestion
                remote_account_person_id=remote_account.extra_data.get(
                    "person_id"
                ),
```

File: cds_ils/ldap/cli.py [hunk 2]
```diff
@@ -12,7 +12,9 @@
 from flask import current_app
 from flask.cli import with_appcontext
 
-from .api import import_ldap_users, sync_users
+from .api import delete_users as ldap_delete_users
+from .api import import_users as ldap_import_users
+from .api import update_users as ldap_update_users
```
Comment (topless) [inline cds_ils/ldap/cli.py:17]:
```suggestion
from .api import (
    delete_users as ldap_delete_users,
    import_users as ldap_import_users,
    update_users as ldap_update_users,
)
```

**Review by topless — APPROVED** [review]

File: cds_ils/ldap/api.py (hunk 1 above)
Comment (ntarocco) [reply cds_ils/ldap/api.py:399]: No, this is on purpose. It should badly fail if not there.

File: cds_ils/ldap/cli.py (hunk 2 above)
Comment (ntarocco) [reply cds_ils/ldap/cli.py:17]: it is `isort` doing, sorry :)

**Review by kprzerwa — APPROVED** [review]

**Pull request merged**